src/libs/actions/BankAccounts.js

```diff
@@ -363,6 +363,16 @@ function openWorkspaceView() {
     API.read('OpenWorkspaceView');
 }
 
+/**
+ * Set the reimbursement account loading so that it happens right away, instead of when the API command is processed.
+ *
+ * @param {Boolean} isLoading
+ * @return {Promise}
+ */
+function setReimbursementAccountLoading(isLoading) {
+    return Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {isLoading});
```

Comment:
At first I set it using the optimisticData, however that presented a problem when going offline and then back online on the reimburse expenses page. The `isOffline` prop updates to `false` before the API command is actually processed, which means that the page briefly shows stale data, and then a loading spinner, and then the correct data. Using this action I'm able to set `isLoading` right away.

Reply:
Hmmmm... The only spot I see this function being used is right before we call `openWorkspaceReimburseView` - so how would using `optimisticData` in that API command not work? 🤔

Reply:
I think it's because the Network request queue doesn't process the command right away. I'll make that change and record a video tomorrow, or you could try it if you want.

Reply:
Oh hmmmm.... cause it's a read. So I guess we're getting blocked by a WRITE request that's hanging? Looks like that was added here: https://github.com/Expensify/App/commit/c653d1a6d8a906e68ed779f7cbadd45253b1f1ca

Reply:
Oh interesting, good investigation. Given that, are you happy with the current approach?

Reply:
I have no clue 😅 - maybe it's fixable or there's a way around it. I'll ask in Slack.

Reply:
Asked [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1670612329234789)

Reply:
I added a comment to provide some more explanation.